napari/layers/image/image.py

```diff
@@ -143,7 +154,15 @@ def __init__(
         )
 
         # Set data
+        self.pyramid = pyramid
```

Comment:
I'd call this `is_pyramid`, as otherwise it seems like it holds the pyramid as opposed to e.g. image/data holding the full-res image.

Reply:
should the keyword arg be `is_pyramid` too then? I think so.

Also should it be `is_rgb` instead of `rgb`? I copied that convention here.

Reply:
i've replaced `is_pyramid` everywhere including keyword arg (i like the keyword arg and the property being the same), but i've left `rgb` but can switch to `is_rgb` in a follow up PR. Thoughts @jni?

Reply:
I think RGB is ok. I found self.pyramid confusing because it seems like that should point to the actual pyramid. But I don’t think one would expect self.rgb to hold the rgb image.